src/debug/HyprCtl.cpp

```diff
@@ -920,14 +923,25 @@ std::string systemInfoRequest(eHyprCtlOutputFormat format, std::string request)
 
     result += "\n\n";
 
-#if defined(__DragonFly__) || defined(__FreeBSD__)
-    const std::string GPUINFO = execAndGet("pciconf -lv | fgrep -A4 vga");
-#elif defined(__arm__) || defined(__aarch64__)
-    const std::string GPUINFO = execAndGet("cat /proc/device-tree/soc*/gpu*/compatible");
-#else
-    const std::string GPUINFO = execAndGet("lspci -vnn | grep VGA");
-#endif
-    result += "GPU information: \n" + GPUINFO;
+    int fd = open("/dev/dri/card0", O_RDONLY | O_CLOEXEC);
```

Comment:
this is dumb, we may not even be running on card0. You should get the card(s) path(s) from the current aq session g_pCompositor->m_pAqBackend->session